tool/tctl/common/collection.go

```diff
@@ -1464,3 +1465,21 @@ func (c *accessListCollection) writeText(w io.Writer, verbose bool) error {
 	_, err := t.AsBuffer().WriteTo(w)
 	return trace.Wrap(err)
 }
+
+type vnetConfigCollection struct {
+	vnetConfig *vnet.VnetConfig
+}
+
+func (c *vnetConfigCollection) resources() []types.Resource {
+	return []types.Resource{types.Resource153ToLegacy(c.vnetConfig)}
+}
+
+func (c *vnetConfigCollection) writeText(w io.Writer, verbose bool) error {
+	var dnsZoneSuffixes []string
+	for _, dnsZone := range c.vnetConfig.Spec.CustomDnsZones {
+		dnsZoneSuffixes = append(dnsZoneSuffixes, dnsZone.Suffix)
+	}
+	fmt.Fprintln(w, "ipv4_cidr_range: ", c.vnetConfig.Spec.Ipv4CidrRange)
+	fmt.Fprintln(w, "custom_dns_zones:", strings.Join(dnsZoneSuffixes, ", "))
```

Comment:
for consistency with the other collections, I would prefer this as a single row table with column headers like we have for `ClusterNetworkingConfig`

Reply:
sure, done